API-Automation/data/commonData.js

```diff
@@ -0,0 +1,161 @@
+export const newUserData = {
+  "id": 2,
+  "username": "Username2",
+  "firstName": "Firstname2",
+  "lastName": "Lastname2",
+  "email": "firstname@gmail.com",
+  "password": "password",
+  "phone": "123456789",
+  "userStatus": 1
+}
+
+export const createUserPath = {
+  "path": "user"
+}
+
+export const getUserPath = {
+  "path": "user/Username2"
+  //this shouldn't be hardcoded, there has to be the way to add this differently
+}
+
+export const createUserWithListData =
+  [
+    {
+      "id": 3,
+      "username": "Username3",
+      "firstName": "Name3",
+      "lastName": "lastName3",
+      "email": "name3@gmail.com",
+      "password": "password3",
+      "phone": "45678976",
+      "userStatus": 1
+    },
+    {
+      "id": 4,
+      "username": "Username4",
+      "firstName": "Name4",
+      "lastName": "lastname4",
+      "email": "name4@gmail.com",
+      "password": "password4",
+      "phone": "0986756879",
+      "userStatus": 0
+    }
+  ]
+
+export const createUserWithListPath = {
+  "path": "user/createWithList"
+  //this shouldn't be hardcoded, there has to be the way to add this differently
```

Comment:
There is another way by creating config files, which will contain all paths, but they are considered as configs and they are hardcoded, so, this is okay, for now, let's keep it like this.